otterai/indexer.py

```diff
@@ -30,8 +30,20 @@ def get_file_type(file_path: str) -> Optional[str]:
         return 'frontend'
     elif ext in ['.sql', '.graphql']:
         return 'data'
-    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go']:
+    elif ext in ['.test.js', '.test.ts', '.spec.py', '_test.go', '.spec.js', '.spec.ts']:
```

Comment:
📝 Good addition! Consider consolidating similar file type checks into a single condition to improve readability. For example, group all test-related extensions together. 📂